Gruntfile.js

```diff
@@ -118,6 +119,23 @@ module.exports = function(grunt) {
         src: ['package.json']
       }
     },
+    'github-release': {
+      options: {
+        repository: 'videojs/videojs-contrib-hls',
+        auth: {
+          user: process.env.VJS_GITHUB_USER,
+          password: process.env.VJS_GITHUB_TOKEN
+        },
+        release: {
+          'tag_name': 'v' + pkg.version,
+          name: pkg.version,
+          body: require('chg').find(pkg.version).changesRaw
+        }
+      },
+      files: {
+        'dist': ['videojs.hls.min.js']
```

Comment:
We should. I was shooting for MVP with this PR because the releaser was a bit finicky. Hopefully it will be easy to add the unminified version to this once we get the process working smoothly.